--- a/src/lib.cpp
+++ b/src/lib.cpp
@@ -52,7 +52,6 @@
 #include <sys/stat.h>
 #include <dirent.h>
 #include <locale.h>
-#include <libintl.h>
 #include <limits>
 #include <math.h>
 #include <ncurses.h>
--- a/src/lib.h
+++ b/src/lib.h
@@ -25,7 +25,6 @@
 #ifndef INCLUDE_GUARD_LIB_H
 #define INCLUDE_GUARD_LIB_H
 
-#include <libintl.h>
 #include <stdint.h>
 
 /* Include only for Automake builds */
@@ -34,6 +33,10 @@
 #endif
 
 #ifdef ENABLE_NLS
+#include <libintl.h>
+#endif
+
+#ifdef ENABLE_NLS
 #define _(STRING)    gettext(STRING)
 #else
 #define _(STRING)    (STRING)
